frontend/src/views/admin/SubscriberPanelView.vue

```diff
@@ -14,6 +14,9 @@
       {{ pageError }}
     </alert-box>
   </div>
+  <admin-nav>
+
+  </admin-nav>
```

Comment:
And same here: Is there supposed to be a child element here or can we use a stand-alone tag?